concepts/05 UI Components/DataGrid/10 Enhance Performance on Large Datasets/030 Rendering Optimization.md

```diff
@@ -1,7 +1,16 @@
-The UI component renders all rows and columns once the data is loaded. To improve performance, set the [rowRenderingMode](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/scrolling/rowRenderingMode.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/scrolling/#rowRenderingMode') and [columnRenderingMode](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/scrolling/columnRenderingMode.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/scrolling/#columnRenderingMode') properties in the **scrolling** object to *"virtual"*. These properties specify that UI elements are only rendered when they come into the viewport.
+If you work with a large data set, apply the following recommendations to optimize rendering performance:
```

Comment:
The phrase "apply recommendation" sounds strange to me. I don't think these words are commonly used together. I'd say "follow recommendations", maybe "consider recommendations". 

```suggestion
If you work with a large data set, follow the recommendations below to optimize rendering performance:
```